### PR TITLE
[RHACS] Added removing central-attached PV instructions

### PR DESCRIPTION
**Pull request by gaurav-nelson**
For https://issues.redhat.com/browse/ROX-17651

Preview:
1. https://61082--docspreview.netlify.app/openshift-acs/latest/upgrading/upgrade-operator.html#remove-central-attached-pv-overview_upgrade-operator
2. https://61082--docspreview.netlify.app/openshift-acs/latest/upgrading/upgrade-helm.html#remove-central-attached-pv-overview_upgrade-helm
3. https://61082--docspreview.netlify.app/openshift-acs/latest/upgrading/upgrade-roxctl.html#remove-central-attached-pv-overview_upgrade-roxctl

Cherry-pick:
- `rhacs-docs-4.1`

### CODE REVIEW

**Review by dashrews78 — APPROVED** [review]
LGTM

File: modules/remove-central-attached-pv-overview.adoc [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+//
+// * upgrading/upgrade-operator.adoc
+:_content-type: CONCEPT
+[id="remove-central-attached-pv-overview_{context}"]
+= Remove Central-attached PV
+
+[role="_abstract"]
+Kubernetes and {ocp} do not delete persistent volumes (PV) automatically. When you upgrade {product-title-short} from earlier versions, the Central PV called `stackrox-db` remains mounted. However, in {product-title-short} 4.1, Central does not need the previously attached PV anymore.
+
+The PV has data and persistent files used by earlier {product-title-short} versions. You can use the PV to roll back to an earlier version before {product-title-short} 4.1. Or, if you have a large RocksDB Central backup bundle, you can use the PV to restore that data.
```
Comment (tmalove) [inline modules/remove-central-attached-pv-overview.adoc:11]:
```suggestion
The PV has data and persistent files used by earlier {product-title-short} versions. You can use the PV to roll back to an earlier version before {product-title-short} 4.1. Or, if you have a large `RocksDB Central` backup bundle, you can use the PV to restore that data.
```
See https://redhat-documentation.github.io/asciidoc-markup-conventions/ (suggest global change)

File: modules/remove-central-attached-pv-overview.adoc [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+If you do not plan to roll back or restore from legacy RocksDB backups, you can remove the Central-attached persistent volume claim (PVC) to free up the storage.
```
Comment (tmalove) [inline modules/remove-central-attached-pv-overview.adoc:13]:
```suggestion
If you do not plan to roll back or restore from earlier `RocksDB` backups, you can remove the Central-attached persistent volume claim (PVC) to free up the storage.
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#l

File: modules/remove-central-attached-pv-overview.adoc (hunk 1 above)
Comment (gaurav-nelson) [reply modules/remove-central-attached-pv-overview.adoc:11]: [RocksDB](https://rocksdb.org/) is a product and therefore I want to keep the name as it is without the backticks. However, I've updated the sentence for more clarity:
```
... a large RocksDB backup bundle for Central, ...
```

**Pull request merged**